charts/argo-cd/values.yaml

```
@@ -175,6 +175,10 @@ configs:
     # -- The name of tracking label used by Argo CD for resource pruning
     application.instanceLabelKey: argocd.argoproj.io/instance
 
+    # -- Enable control of the service account used for the sync operation
```

According to the doc, it's an alpha feature, so we want to put `alpha` .
```suggestion
    # -- Enable control of the service account used for the sync operation (alpha)
```

---

Good suggestion :)